src/lib/config.js

```diff
@@ -162,6 +162,16 @@ if (isDefinedConfigValue(process.env.SERVICES_RETRIES)) {
 if (isDefinedConfigValue(process.env.SERVICES_TIME)) {
   initialConfig['services.time'] = process.env.SERVICES_TIME
 }
+// Set composr-core execution phrases with Node VM
+if (isDefinedConfigValue(process.env.EXEC_WITH_VM)) {
+  initialConfig['execution.vm'] = process.env.EXEC_WITH_VM
```

Comment:
Please add a JSON.parse(provess.env.EXEC_WITH_VM) to enforce that the value is a boolean

Also `execution.vm` should be defined in the config.json as false